### DISCUSSION TITLE
Detach REPL from UART0 on RP2 board

### DISCUSSION THREAD

**Discussion by arbego** [micropython/micropython#9820] [Hardware & Peripherals] [answered]
I'd like to detach REPL console from UART0, so I can use the UART0 for my own purposes. Is this possible on RP2 board? On esp board I can do `uos.dupterm(None, 1) # disable REPL on UART(0)`. But on RP2 board I get error message: `AttributeError: 'module' object has no attribute 'dupterm'`.

**Comment by robert-hh** [comment, marked as answer]
The pico rp2 has by default REPL at the USB port, not at UART. So you can use UART 0 as such.

**Reply by arbego** [reply]
Thanks for making this clear.
